tests/stylefmt/values/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -16,8 +16,8 @@ exports[`values.css 1`] = `
   border: 1px solid rgba(0, 0, 0, .3);
   font-family: Arial, sans-serif;
   color: rgba(0, 0, 0, 1);
-  margin: 0 20px 0 -24px;
-  background-position: 20% -26px;
+  margin: 0 20px 0 - 24px;
```

Comment:
well `0 20px 0 - 24px;` isn't valid there, because that's a calculation outside of `calc()` but the line as it was previously `0 20px 0 -24px;` is valid, because that's considered a sign for the number.